### PR TITLE
Add firewall control feature

### PR DESCRIPTION
**Pull request by ebenzecri**

### CODE REVIEW

File: ansible-scylla-node/tasks/common.yml [hunk 1]
```diff
@@ -277,3 +277,20 @@
   become: true
   when: update_sysctl|bool is defined and update_sysctl|bool
 
+- name: Deactivate firewall
+  block:
+    - name: Deactivate front-end firewall services
+      service:
+        name: "{{ item }}"
+        state: stopped
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/common.yml:285]: what will happen if the machine will be restarted?
will the firewalls start back up?
what if ufw is not installed? (you have ignore errors, but I don't think this is a good solution, if you ignore error, then later you will figure out the firewall is not down and your connection doesn't work)

File: ansible-scylla-node/tasks/common.yml [hunk 2]
```diff
@@ -277,3 +277,20 @@
   become: true
   when: update_sysctl|bool is defined and update_sysctl|bool
 
+- name: Deactivate firewall
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/common.yml:280]: I'd still be happy if you could test the whole deploy and ideally even with a 3 node cluster (I will try to merge the dev branch with molecule tests asap).
Why?
Because the task that checks if cluster is up is far far before this task and it would fail, since the firewall is up back then.
That said I think this task should be properly split to both RHEL and Debian yml files and do proper stuff there without ignore-ing the errors.
The this file will work as charm, since cluster will be created.

**Comment by ebenzecri** [comment]
@tarzanek I've improved the code, please take a look. Thanks!

File: ansible-scylla-node/tasks/common.yml [hunk 3]
```diff
@@ -277,3 +296,20 @@
...
+      loop:
+        - firewalld
+        - iptables_services
+        - ufw
+    - name: Flush all iptables rules
+      iptables:
+        flush: yes    
+  when: not firewall_enabled
+  ignore_errors: yes
+  become: true
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/common.yml:315]: the duplication of task is on purpose or forgotten from previous commit?

File: ansible-scylla-node/tasks/main.yml [hunk 4]
```diff
@@ -1,5 +1,8 @@
 ---
 
+- name: Populate service facts
```
Comment (tarzanek) [inline ansible-scylla-node/tasks/main.yml:3]: why is this needed?

Comment (tarzanek) [reply ansible-scylla-node/tasks/main.yml:3]: ah, I see now, only if service is in facts, okay

**Comment by tarzanek** [comment]
sorry for late review, so just clean up duplication and I think this can be merged
tia @ebenzecri !

File: ansible-scylla-node/tasks/common.yml (hunk 3 above)
Comment (ebenzecri) [reply ansible-scylla-node/tasks/common.yml:315]: @tarzanek Ignore this commit and use the last one. Sorry for the hassle.

**Comment by ebenzecri** [comment]
@tarzanek Ignore this commit and use the last one which has all the corrections you pointed. Sorry for the hassle.

**Comment by ebenzecri** [comment]
@tarzanek All done.

**Comment by tarzanek** [comment]
thank you @ebenzecri :-) ! merging
